reading.Rmd

```diff
@@ -1095,23 +1098,26 @@ of accessing data through an API in this book, with the hope that it gives you e
 idea that you can learn how to use another API if needed.
 
 In particular, in this book we will show you the basics of how to use
-the [rtweet](https://github.com/ropensci/rtweet) package in R
+the `rtweet` package in R
 \index{API!rtweet}\index{rtweet}\index{Twitter} 
-to access data from the [Twitter](www.twitter.com) API. One nice feature of this particular 
+to access data from the Twitter API. One nice feature of this particular 
 API is that you don't need  a special *token* \index{API!token} to access it; you simply need to 
 make an account with them. Your access to the data will then be authenticated and controlled through
-your account username and password. If you have a Twitter account already (or are willing to make one), you can follow
+your account username and password. If you have a Twitter 
+account already (or are willing to make one), you can follow
 along with the examples that we show here. To get started, load the `rtweet` package:
 
 ```r
 library(rtweet)
 ```
 
-This package provides an extensive set of functions to search Twitter for tweets, users, their followers, and more. 
-Let's construct a small data set of the last 400 tweets and retweets from the [Tidyverse account]((https://twitter.com/tidyverse?lang=en)). A few of the most recent tweets
+This package provides an extensive set of functions to search 
+Twitter for tweets, users, their followers, and more. 
+Let's construct a small data set of the last 400 tweets and 
+retweets from the \@tidyverse account. A few of the most recent tweets
```

Comment:
huge space after this paragraph before Fig 2.8

Reply:
Nothing we can or should do about this (and other spacing-related things like this) until CRC helps us out; it's a LaTeX formatting issue.